--- google-gadgets-for-linux-0.11.0.orig/configure.ac	2009-05-31 07:06:05.000000000 +0200
+++ google-gadgets-for-linux-0.11.0/configure.ac	2009-06-02 11:17:51.495129255 +0200
@@ -1009,6 +1009,7 @@
     GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/widget"
     GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/gtkembedmoz"
     GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/xpcom"
+    GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/js"
     GTKMOZEMBED_CFLAGS="$GTKMOZEMBED_CFLAGS -I$EMBED_INCDIR/string"
     GTKMOZEMBED_LDFLAGS=""
     MOZILLA_FIVE_HOME=`$PKG_CONFIG --variable=libdir $has_gtkmozembed`
@@ -1040,6 +1041,9 @@
         LIBMOZJS_CFLAGS="$LIBMOZJS_CFLAGS -I$LIBMOZJS_INCDIR/unstable"
       fi
     fi
+    if test -f $LIBMOZJS_INCDIR/js/jsapi.h; then
+      LIBMOZJS_CFLAGS="$LIBMOZJS_CFLAGS -I$LIBMOZJS_INCDIR/js"
+    fi
 
     # Find out the real path of libmozjs.so
     LIBMOZJS_LIBDIR=`$PKG_CONFIG --variable=libdir $has_libmozjs`
